typings/systemjs.d.ts

```diff
@@ -0,0 +1,16 @@
+interface System {
+  import(name: string): any;
+  defined: any;
+  amdDefine: () => void;
+  amdRequire: () => void;
+  baseURL: string;
+  paths: { [key: string]: string };
+  meta: { [key: string]: Object };
```

Comment:
This type definitions from the TSD package manager, however now I see that it should be updated. I'll do it tomorrow.